src/Sulu/Bundle/MediaBundle/Resources/public/js/services/format-manager.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/util'], function(Util) {
+
+    'use strict';
+
+    var urls = {
+        formats: '/admin/api/media/%id/formats?locale=%locale',
+        format: '/admin/api/media/%id/formats/%key?locale=%locale'
+    };
+
+    return {
+
+        /**
+         * Loads all formats for a given media and a given locale.
+         *
+         * @param {Number} mediaId
+         * @param {String} locale
+         *
+         * @returns {Object} a promise which gets resolved when the formats have been loaded
+         */
+        loadFormats: function(mediaId, locale) {
+            return Util.load(urls.formats.replace('%id', mediaId).replace('%locale', locale));
```

Comment:
We have a `sprintf` function included in husky, I would use that one instead of replace.